server/controllers/stock/reports/stock/exit/exitToDepot.js

```diff
@@ -0,0 +1,38 @@
+const db = require('../../../../../lib/db');
+
+/**
+ * @function fetch
+ * @description fetch stock exit for patient
+ */
+function fetch(depotUuid, dateFrom, dateTo, showDetails) {
+  const IS_EXIT = 1;
+  const EXIT_TO_DEPOT_ID = 8;
```

Comment:
Both of these `const` could go outside the `fetch()` function, since they don't depend on the function itself.